wandb/plots/utils.py

```diff
@@ -0,0 +1,120 @@
+import wandb
+from wandb import util
+
+# FIXME: look at wandb.util.is_numpy_array() wandb.util.is_pandas* for examples
+np = util.get_module("numpy", required="Logging plots requires numpy")
+pd = util.get_module("pandas", required="Logging dataframes requires pandas")
+scipy = util.get_module("scipy", required="Logging scipy matrices requires scipy")
+spacy = util.get_module("spacy", required="Logging NER and POD requires spacy")
+sklearn = util.get_module("sklearn")
+collections = util.get_module("collections",
+                    required="Logging python iterables requires collections")
```

Comment:
This is a built in but it's a little tricky.  See: https://github.com/wandb/client/blob/master/wandb/wandb_config.py#L5-L8 You'll want to import Sequence and Iterable directly.

Reply:
fixed [here](https://github.com/wandb/client/pull/909/commits/5095e706af4f7346d0cfdc8d33736d8f32f801f6)